src/main/java/org/spongepowered/api/event/entity/player/PlayerEvent.java

```diff
@@ -38,18 +38,6 @@
      *
      * @return The {@link Player} involved
      */
-    Player getPlayer();
-
-    @Override
-    Player getHuman();
-
-    @Override
-    Player getLiving();
-
     @Override
     Player getEntity();
-
-    @Override
-    Player getUser();
```

Comment:
What about `UserEvent`? A `Player` is a subclass of `User`, so there should be an override for that.